src/app/host/host-page/host-page.component.ts

```diff
@@ -31,23 +32,12 @@ export class HostPageComponent implements OnInit {
 
       this.reviewService.getReviews(+this.id).subscribe(reviews => {
         this.reviews = reviews;
-        this.calculateAverageRating();
       });
 
       this.accountService.getAccount(this.id).subscribe(host => {
+        this.host = host;
         this.header = `Host reviews for ${host.name} ${host.surname}`;
       });
     });
   }
-
-  calculateAverageRating() {
-    if (!this.reviews || this.reviews.length <= 0) {
-      this.averageRating = undefined;
-      return;
-    }
-
-    //TODO: Get average rating for host from server (User story 5.3)
-    this.averageRating = this.reviews.map(review => review.rating).reduce((a, b) => a + b) / this.reviews.length;
```

Comment:
I recommend changing these lines so they fetch the average rating again and not deleting this method.